src/trans/gpu/internal/fsc_mod.F90

```diff
@@ -116,9 +120,9 @@ SUBROUTINE FSC(ALLOCATOR,HFSC,PREEL_COMPLEX, KF_FS, KF_UV, KF_SCALARS, KUV_OFFSE
 !*       1.1      U AND V.
 #ifdef OMPGPU
 !$OMP TARGET TEAMS DISTRIBUTE PARALLEL DO COLLAPSE(3) DEFAULT(NONE) &
-!$OMP& PRIVATE(IGLG,IOFF_LAT,IOFF_UV,ZACHTE2) &
-!$OMP& SHARED(IBEG,IEND,IINC,KF_UV,R,OFFSET_VAR,G,D,KF_FS,KUV_OFFSET,F,PREEL_COMPLEX) &
-!$OMP& MAP(TO:IBEG,IEND,IINC,KF_UV,OFFSET_VAR,KF_FS,KUV_OFFSET,ZACHTE2)
```

Comment:
With the removal of `MAP`, how to we ensure `IBEG` etc. are mapped from host to device properly?

Reply:
You're right, I believe I mistakenly assumed OMP's `FIRSTPRIVATE` here handles implicit mapping of variables like OpenACC's `FIRSTPRIVATE`. After reviewing documentation, explicit maps will likely be required. I will address this.